### PR TITLE
feat: added module

### PR DESCRIPTION
**Pull request by owlleg6**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+resource "google_storage_bucket_object" "startup_scripts" {
+  for_each = toset(var.scripts)
+  name     = "templates/${each.key}"
+  bucket   = google_storage_bucket.source.name
+  content  = templatefile("${path.module}/templates/${each.key}", merge(var.scripts_vars, { BUCKET_NAME = google_storage_bucket.source.url }))
+}
+
+resource "google_compute_instance" "bastion" {
+  name           = "${var.env}-${var.product_base_name}-vm-instance"
+  machine_type   = "e2-small"
```
Comment (owlleg6) [inline main.tf]: move to var

File: main.tf [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  can_ip_forward = "false"
```
Comment (owlleg6) [inline main.tf]: move to var

File: main.tf [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    }
+    subnetwork = var.subnet
+  }
+  service_account {
+    email  = google_service_account.bastion_sa.email
+    scopes = var.scopes
+  }
+  allow_stopping_for_update = true
+
+  labels = {
```
Comment (owlleg6) [inline main.tf]: tags to base

File: main.tf [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+  project  = var.project_id
+  name     = "${var.env}-${var.product_base_name}-gcpconsol-ingress-allow"
+  network  = var.network
+  priority = "100"
+  allow {
+    protocol = "tcp"
+    ports    = ["22"]
+  }
+  direction     = "INGRESS"
+  source_ranges = ["35.235.240.0/20"]
```
Comment (owlleg6) [inline main.tf]: move to var, and provide description

File: main.tf (hunk 1 above)
Comment (ivasm-ko) [reply main.tf]: done

File: main.tf (hunk 2 above)
Comment (ivasm-ko) [reply main.tf]: done

File: main.tf (hunk 3 above)
Comment (ivasm-ko) [reply main.tf]: done

File: main.tf (hunk 4 above)
Comment (ivasm-ko) [reply main.tf]: done

**Pull request merged**